packages/cli/src/lib/packager/createDistWorkspace.ts

```diff
@@ -226,7 +228,7 @@ export async function createDistWorkspace(
         portable: true,
         noMtime: true,
         gzip: options.skeleton.endsWith('.gz'),
-      } as CreateOptions & { noMtime: boolean },
+      } as CreateOptions & FileOptions & { noMtime: boolean },
```

Comment:
This change seems unnecessary? 😅

Reply:
With `FileOptions` added static analysis toolkit we use (sonarqube) picked method from types, that return a promise, and did not complain about await on non-promise: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/tar/index.d.ts#L694